sdk/jobs/automl-standalone-jobs/automl-image-classification-multiclass-task-fridge-items/automl-image-classification-multiclass-task-fridge-items.ipynb

```diff
@@ -297,6 +297,34 @@
     "# my_validation_data_input = Input(type=AssetTypes.MLTABLE, path=\"azureml://datastores/workspaceblobstore/paths/vision-classification/valid\")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To create data input from TabularDataset created using V1 sdk, specify the `type` as `AssetTypes.MLTABLE`, `mode` as `InputOutputModes.DIRECT` and `path` in the following format."
```

Comment:
maybe this line can be - `path` in the following format `"azureml:<tabulardataset_name>:<version>"`
and then in the code example we can have `"azureml:fridgeObjectsTrainingDataset:1"` and/or `"azureml:fridgeObjectsTrainingDataset:latest"`

Reply:
Thanks for the suggestion. Done